<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Friend</class>
 <widget class="QWidget" name="Friend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_1">
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color: #F2F2F2;
border-top: 0px;
border-right: 1px solid rgb(130, 130, 130, 50);
border-left: 1px solid rgb(130, 130, 130, 50);
border-bottom: 0px;
}

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="m_publicChatLB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #F2F2F2;
border-top: 0px;
border-right: 0px;
border-left: 0px;
border-bottom: 0px;</string>
          </property>
          <property name="text">
           <string>公共聊天：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color: #F2F2F2;
border-top: 0px;
border-right: 1px solid rgb(130, 130, 130, 50);
border-left: 1px solid rgb(130, 130, 130, 50);
border-bottom: 1px solid rgb(130, 130, 130, 50);
}

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="m_pShowMsgTE">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #F2F2F2;
border: 10px solid #F2F2F2;
</string>
          </property>
          <property name="undoRedoEnabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_5">
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color: #F2F2F2;
border-top: 0px;
border-right: 1px solid rgb(130, 130, 130, 50);
border-left: 1px solid rgb(130, 130, 130, 50);
border-bottom: 0px;
}

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTextEdit" name="m_pInputMsgTE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #F2F2F2;
border: 10px solid #F2F2F2;
</string>
          </property>
          <property name="undoRedoEnabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'微软雅黑'; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">background-color: #F2F2F2;
border: 0px;
border-left: 1px solid rgb(130, 130, 130, 50);
border-right: 1px solid rgb(130, 130, 130, 50);
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>11</number>
        </property>
        <property name="topMargin">
         <number>11</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="m_pMsgSendPB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>11</number>
        </property>
        <item>
         <widget class="QLabel" name="m_usrListLB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>好友：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="m_pFriendListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>360</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
         <weight>3</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QListView::item {
    height: 35px;
}
QListView {
	font: 25 12pt &quot;Consolas&quot;;
	border: 10px solid white;
}

QListView::item:hover {
	background-color: transparent;
	padding: 5px;
	border-left: 5px solid rgb(130, 130, 130);
}
QListView::item:selected {
	background-color: transparent;
	color: black;
	padding: 5px;
	border-left: 5px solid black;
}

QListWidget{
	border-bottom: 1px solid rgb(130, 130, 130, 20);
}

QListWidget::item:focus {
    outline: none;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pFlushFriendPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 0px;
background-color: rgba(255, 255, 255, 1);
border-bottom: 2px solid rgb(130, 130, 130, 20);
}

QPushButton:hover{
border: 0px;
background-color: rgba(242, 242, 242, 0.1);
}

QPushButton:pressed{
border: 0px;
background-color: rgba(255, 255, 255, 1);
}
</string>
       </property>
       <property name="text">
        <string>刷新好友列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pShowOnlineUsrPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 0px;
background-color: rgba(255, 255, 255, 1);
border-bottom: 2px solid rgb(130, 130, 130, 20);
}

QPushButton:hover{
border: 0px;
background-color: rgba(242, 242, 242, 0.1);
}

QPushButton:pressed{
border: 0px;
background-color: rgba(255, 255, 255, 1);
}
</string>
       </property>
       <property name="text">
        <string>显示在线用户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pSearchUsrPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 0px;
background-color: rgba(255, 255, 255, 1);
border-bottom: 2px solid rgb(130, 130, 130, 20);
}

QPushButton:hover{
border: 0px;
background-color: rgba(242, 242, 242, 0.1);
}

QPushButton:pressed{
border: 0px;
background-color: rgba(255, 255, 255, 1);
}
</string>
       </property>
       <property name="text">
        <string>查找用户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pPrivateChatPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 0px;
background-color: rgba(255, 255, 255, 1);
border-bottom: 2px solid rgb(130, 130, 130, 20);
}

QPushButton:hover{
border: 0px;
background-color: rgba(242, 242, 242, 0.1);
}

QPushButton:pressed{
border: 0px;
background-color: rgba(255, 255, 255, 1);
}
</string>
       </property>
       <property name="text">
        <string>私聊</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pAddFriend_PB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 0px;
background-color: rgba(255, 255, 255, 1);
border-bottom: 2px solid rgb(130, 130, 130, 20);
}

QPushButton:hover{
border: 0px;
background-color: rgba(242, 242, 242, 0.1);
}

QPushButton:pressed{
border: 0px;
background-color: rgba(255, 255, 255, 1);
}
</string>
       </property>
       <property name="text">
        <string>加好友</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pDelFriendPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 0px;
background-color: rgba(255, 255, 255, 1);
border-bottom: 2px solid rgb(130, 130, 130, 20);
}

QPushButton:hover{
border: 0px;
background-color: rgba(242, 242, 242, 0.1);
}

QPushButton:pressed{
border: 0px;
background-color: rgba(255, 255, 255, 1);
}
</string>
       </property>
       <property name="text">
        <string>删除好友</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
